### PR TITLE
Allow custom LayoutBlockBuilder to be configured in the MarkdownTextView

### PR DESCRIPTION
**Pull request by jvanzummeren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,12 +262,28 @@ Add the rule to MarkyMark
...
+markdownTextView.add(rule: MyCustomRule())
+```
+
 Add the block builder to your layout converter
 
 ```swift
 converter.addLayoutBlockBuilder(MyCustomLayoutBlockBuilder())
 ```
 
+Or when using the MarkdownTextView use either of these options (depending on the configuration view or attrributedString):
```
Comment (edwinveger) [inline README.md:277]: Small typo here (`attrributedString`).

File: markymark/Classes/Default implementations/MarkDownTextView.swift [hunk 2]
```diff
@@ -90,6 +90,14 @@ open class MarkDownTextView: UIView {
         markyMark.addRule(rule)
     }
 
+    public func addViewLayoutBlockBuilder(layoutBlockBuilder: LayoutBlockBuilder<UIView>) {
```
Comment (edwinveger) [inline markymark/Classes/Default implementations/MarkDownTextView.swift:93]: We could use `_` as argument label to shorten the call site. The repetition of `layoutBlockBuilder` is too verbose for my taste. Same for method below.

**Review by edwinveger — COMMENTED** [review]
Few syntax comments.

**Review by Basca — COMMENTED** [review]
I have no other remarks than what Edwin has already provided.

**Review by edwinveger — APPROVED** [review]

**Review by Basca — APPROVED** [review]

**Review by murilxaraujo — APPROVED** [review]

**Pull request merged**